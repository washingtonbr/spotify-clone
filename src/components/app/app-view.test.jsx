import React from 'react';
import renderer from 'react-test-renderer';
import AppView from './app-view';

describe('AppView', () => {
  test('render corectly', () => {
    const tree = renderer
      .create(<AppView />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
