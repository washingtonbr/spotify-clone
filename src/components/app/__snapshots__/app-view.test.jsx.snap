// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppView renders correctly 1`] = `
<div
  className="sc-fznMAR cjBjob"
>
  <div
    className="sc-fznyAO gevFZD"
  >
    <div
      className="sc-fznKkj gHuSSi"
    >
      <img
        alt="Spotify"
        className="sc-fznZeY eXznpz"
        src="test-file-stub"
      />
      <h1
        className="sc-fzokOt jQYRUm"
      >
        Pesquise, favorite e ou√ßa!
      </h1>
      <button
        className="sc-fzqBZW htnOaW"
        onClick={[Function]}
      >
        Conecte-se via Spotify
      </button>
    </div>
  </div>
</div>
`;
